src/Form/DataTransformer/QuerySearchOptionsTransformer.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EMS\CoreBundle\Form\DataTransformer;
+
+use Symfony\Component\Form\DataTransformerInterface;
+
+final class QuerySearchOptionsTransformer implements DataTransformerInterface
+{
+    public function transform($value)
+    {
+        $searchConfig = $this->jsonFormat($value, 'searchConfig');
+        $query = $this->jsonFormat($value, 'query');
+
+        return [
+            'searchConfig' => $searchConfig,
+            'query' => $query,
+        ];
```

Comment:
Why a searchConfig? The query is not enough?

Reply:
It is deleted in my last commit, it should not be there anymore.